### PR TITLE
histogram_positive_slope

### PR DESCRIPTION
**Pull request by JakubMeixner**

### CODE REVIEW

**Review by ggruszczynski — CHANGES_REQUESTED** [review]

File: numerical_solvers/visualization/histogram_turbulence.py
[the remark this answers is not in the document]
Comment (ggruszczynski) [reply numerical_solvers/visualization/histogram_turbulence.py:1]: ok, so lets leave it as it is

**Comment by JakubMeixner** [comment]
added  linear positive increasing spectrum in file /home/computergraphics/Documents/jjmeixner/generative-inverse-heat-dissipation/numerical_solvers/solvers/1D_spectral_turbulence_generator.py


image

File: output/iteration_0.jpg
Comment (ggruszczynski) [inline output/iteration_0.jpg:1]: remove img

**Review by ggruszczynski — APPROVED** [review]
lgtm (looks good to me)

**Pull request merged**